RS/RS-Project15/DictionaryImplementation/DictionaryImplementationTest.java

```diff
@@ -0,0 +1,88 @@
+package dictionarymanagement;
+
+import static org.junit.Assert.*;
+
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+import java.util.*;
+
+public class DictionaryImplementationTest {
+    public final String FILENAME = "C:\\Users\\user39\\Downloads\\Meaning.json";
+    DictionaryImplementation operation;
+
+    @Before
+    public void testAddDictionaryEntry() {
+        operation = new DictionaryImplementation();
+        operation.addDictionaryEntry(FILENAME);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void testGetValueOfKey() {
```

Comment:
this test method is to deleteDictionaryEntry but the method name is not aligned. please update